src/main/flight/imu.c

```diff
@@ -361,10 +361,10 @@ static bool imuIsAccelerometerHealthy(float *accAverage)
         accMagnitude += a * a;
     }
 
-    accMagnitude = accMagnitude * 100 / (sq((int32_t)acc.dev.acc_1G));
+    accMagnitude = sqrtf(accMagnitude) * acc.dev.acc_1G_rec;
 
-    // Accept accel readings only in range 0.90g - 1.10g
-    return (81 < accMagnitude) && (accMagnitude < 121);
+    // Accept accel readings only in range 0.80g - 1.20g
```

Comment:
This did change accepter range from 0.9-1.1 to 0.8-1.2 .. this change is probably desirable, but it is undocumented ... 

(old code did test square of accMagnitude against square of limits to avoid sqrtf call)

Reply:
I assumed the range was a little too tight and expanded it, but we can change it back.

the problem of the old code math is that it doesn't calculate the magnitude, AFAIK there's no way to calculate the magnitude of a vector without square root, and if you simplify the math that was there, it would be same as a simple sum of x, y and z / acc_1g (the 'squaring' was doing nothing)

In hindsight, a better optimization would be to simply remove the squares and square root altogether and leave it as a simple sum of x, y and z * acc_1g_rec and use 0.64 < mag < 1.44 (or the old 81 < mag < 121)

Reply:
`a < b` is equivalent to `a^2 < b^2` (when both a and b are positive) - original code did square the limits (constants) instead of computing square root of squared magnitude ... 

```
accMagnitude = accMagnitude * sq(acc.dev.acc_1G_rec);  // squared magnitude in g

return sq(.8) < accMagnitude && accMagnitude < sq(1.1);

```
renaming `accMagnitude` to `accMagnitudeSq` will help a lot with readability ..